### ISSUE TITLE
Getting Started with the Lighting Design System: Missing file in SLDS Zip

### ISSUE THREAD

**Issue by qkershner** [salesforce-ux/design-system#302]
The **Getting Started with the Lighting Design System** module requires a download of the Lighting Design System zip file, upload to a static resource, and a change to the Visual Force markup to reference the appropriate asset.  However, Lightning Design System version 2.0.3 doesn't appear to have a _salesforce-lightning-design-system-vf.min.css_ file as is identified and required by the module?  I noted other modules recently that have not been updated to address changes to the current version of the SLDS zip (file structure and naming changes).  In those cases, it was easy enough to modify the example markup to connect to the appropriate file.  But in this case, the _"...-vf.min.css"_ file does not exist in the file structure.

Please advise?

**Comment by kaelig** [comment]
Glad you are looking to learn about the Lightning Design System, @qkershner! And thanks for getting in touch as well.

[The beginning of the second step of the module](https://trailhead.salesforce.com/en/lightning_design_system/lightning-design-system2) describes how to download the design system and install it into Visualforce. Have you tried following these instructions?

**Comment by qkershner** [comment]
I have followed the instructions to the letter (and have multiple times over the course of the trailhead modules).  However, as I described, the file referenced in the markup does not exist within the newest versions (at least within 2.0.2 and 2.0.3) of the downloadable zip-file.  The developer.salesforce.com community has confirmed that the file structure has changed and that all trailhead modules may need to have the SLDS markup audited for accuracy.
Best, 

Squire Kershner, Nerdcave Consulting 
Salesforce Certified Advanced Administrator, Force.com Developer,
Platform App Builder and Developer, Sales Cloud Consultant




On Fri, Aug 26, 2016 at 8:38 AM -0500, "Kaelig Deloumeau-Prigent" <notifications@github.com> wrote:












Glad you are looking to learn about the Lightning Design System, @qkershner! And thanks for getting in touch as well.



The beginning of the second step of the module describes how to download the design system and install it into Visualforce. Have you tried following these instructions?

**Comment by kaelig** [comment]
Thank you for confirming this. We recently updated the trailhead modules to reflect those changes, and created a CSS Customizer tool to generate those -vf styles.

Have you tried downloading the zip file using the CSS Customizer, or have you downloaded the zip from another source?

**Comment by qkershner** [comment]
That would explain things.  The CSS Customizer wouldn't run/download anything so I tried to move forward without it.  If that's the piece that was missing, my apologies for the confusion.
Squire 

   

 On Friday, August 26, 2016 1:57 PM, Kaelig Deloumeau-Prigent <notifications@github.com> wrote:
 

 Thank you for confirming this. We recently updated the trailhead modules to reflect those changes, and created a CSS Customizer tool to generate those -vf styles.Have you tried downloading the zip file using the CSS Customizer, or have you downloaded the zip from another source?—
